editor/src/components/navigator/left-pane/github-pane/index.tsx

```diff
@@ -358,16 +407,16 @@ const BranchBlock = () => {
   )
 }
 
-const PullIcon = () => {
+const PullIcon = ({ fill }: { fill?: string }) => {
```

Comment:
This change is going to conflict with changes from #2901

Reply:
huh - alright, let's merge #2901 and resolve here